plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/ScriptSelectorPanel.java

```diff
@@ -63,6 +65,7 @@
  * Script selector panel (shell)
  */
 public class ScriptSelectorPanel extends AbstractPopupPanel {
+    public static final String SCRIPT_DIALOG_MODE = "script_dialog_mode";
```

Comment:
1. Preference key constants must start with `PREF_`.
2. Preference key must be unique, prefixed with some namespace, especially in the case of the global preference store.
2. This preference stores a boolean. It's not clear what it means in the context of a _mode_. What's `mode=true`?
3. This preference has no default value associated with it. You must define it in the plugin initializer.